Honour CATKIN_PREFIX_PATH at runtime instead of ROS' hack to use CMAKE_PREFIX_PATH.

Index: pluginlib-1.11.2/include/pluginlib/class_loader_imp.hpp
===================================================================
--- pluginlib-1.11.2.orig/include/pluginlib/class_loader_imp.hpp
+++ pluginlib-1.11.2/include/pluginlib/class_loader_imp.hpp
@@ -68,7 +68,7 @@ namespace
 std::vector<std::string> catkinFindLib()
 {
   std::vector<std::string> lib_paths;
-  const char * env = std::getenv("CMAKE_PREFIX_PATH");
+  const char * env = std::getenv("CATKIN_PREFIX_PATH");
   if (env) {
     std::string env_catkin_prefix_paths(env);
     std::vector<std::string> catkin_prefix_paths;
